low-level/framework/platforms/realstor/realstor_enclosure.py

```diff
@@ -97,6 +97,7 @@ class RealStorEnclosure(StorageEnclosure):
 
     poll_system_ts = 0
     mc_timeout_counter = 0
+    ws_responce_status = None
```

Comment:
spell check - response

Also i dont see the need for this

Reply:
just the `mc_timeout_counter` is not enough to decide for fault_resolved alert as we are fetching whether fault was previously present and previous fault type from persistent cache, in next run cycle all 3 parameters, mc_timeout_count=0, falut_alert = True, and previous_fault != FAULT_RESOLVED will be true, so even tough connection has failed it will give FAULT_RESOLVED, when fault is not resolved, thus response_status is required.